### DISCUSSION TITLE
Smokeview crashes at startup

### DISCUSSION THREAD

**Discussion by mpachera** [firemodels/fds#12204] [Q&A]
Dear,

I am running a large mode (30 M elements) with slicefiles and 3D plots. However, when I try to open the results in Smokeview I can't load them. When I click on the Smokeview icon the window opend, but it stops at "wrapping up" and then the window closes by itself. I tried to move the Smokeview file somewhere else where there are no results an I can load the geometry, but I can't view my results. I also tried to run from command line Smokeview with the option fast, but I couldn't load the results. Is there a way I can handle this?

Thanks in advance.

Matteo

**Comment by gforney** [comment]
Upload the input file and ill take a look

**Comment by gforney** [comment]
What version of fds and smokeview are you using? Type fds and smokeview -v

**Comment by mpachera** [comment]
The simulation ran with the latest FDS version (SMV-6.8.0-0-gd02fd68-release). 

Please find the input file below, it's quite large because the simulation domain is very large.
[input.txt](https://github.com/firemodels/fds/files/13202880/input.txt)

**Comment by mcgratta** [comment]
This is now an issue #12205
